BREAKINGCHANGES.md

```diff
@@ -538,6 +538,20 @@ If you prefer platform support vote for the https://experience.dynamics.com/idea
 
 **Solution**: Event has been moved to `codeunit 1440 "RC Headlines Page Common"`, function `OnIsAnyExtensionHeadlineVisible`.
 
+**Error**: _'Codeunit "Conf./Personalization Mgt."' does not contain a definition for 'GetCurrentProfileIDNoError'_
+
+**Solution**: Profiles are no longer uniquely identified by their ID: use `'GetCurrentProfileNoError(var AllProfile: Record "All Profile")'` instead.
+
+**Error**: _'Codeunit "Conf./Personalization Mgt."' does not contain a definition for 'ExportProfilesInZipFile'_\
+**Error**: _'Codeunit "Conf./Personalization Mgt."' does not contain a definition for 'ExportProfiles'_\
+**Error**: _'Codeunit "Conf./Personalization Mgt."' does not contain a definition for 'ImportProfiles'_
+
+**Solution**: Export the profiles in the AL format using `DownloadProfileConfigurationPackage` instead.
+
+**Error**: _The event 'OnGetBuiltInRoleCenterFilter' is not found in the target_
+
+**Solution**: Mark the profiles as `AllProfile.Enabled := false` to hide it from the profile selection, or subscribe to the [Page Trigger Events](https://docs.microsoft.com/en-us/dynamics-nav/event-types#PageEvents) to hide the profile from specific pages.
+
 **Error**: _'Codeunit "Type Helper"' does not contain a definition for 'RegexReplace'_
```

Comment:
Looks strange. I believe errors related to Conf./Personalization Mgt should come together.

Reply:
Merged into the BaseApp section :)